docs/source/api/user-guide/measurements/randomized-response.rst

```diff
@@ -354,3 +352,61 @@ Since the dataset size is known, simply post-process the mean estimates:
             >>> print("unbiased categorical count:", estimated_cat_count)
             unbiased categorical count: {'A': ..., 'B': ..., 'C': ..., 'D': ...}
 
+Bit Vector Randomized Response
+------------------------------
+
+Another way to generalize boolean randomized response 
+is by considering bit vectors with limited weight.
+In this setting, neighboring datasets may be completely different bitvectors, 
+but all bitvectors may only have a limited number of true bits (the weight).
+
+.. tab-set::
+
+    .. tab-item:: Python
+        :sync: python
+
+        .. code:: pycon
+
+            >>> import pytest
+            >>> np = pytest.importorskip("numpy")
+
+            >>> # construct the measurement
+            >>> m_rr = dp.m.make_randomized_response_bitvec(
+            ...     dp.bitvector_domain(max_weight=3),
+            ...     dp.discrete_distance(),
+            ...     f=0.8,
+            ... )
+
+            >>> # the de-biaser will expect little endian data
+            >>> data = np.packbits(
+            ...     [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0], # fmt: skip
+            ...     bitorder="little",
+            ... ) # fmt: skip
+
+            >>> # invoke the measurement: np -> bytes -> mechanism -> bytes -> np
+            >>> release = np.frombuffer(m_rr(data), dtype=np.uint8)
+
+            >>> # use unpackbits to visualize the bitvectors
+            >>> print("data:   ", np.unpackbits(data, bitorder="little"))
+            data:    [0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 1 1 0 0]
+            >>> print("release:", np.unpackbits(release, bitorder="little"))   # fmt: skip # doctest: +SKIP
+            release: [1 0 0 0 0 1 1 1 1 0 0 1 1 1 1 0 0 1 1 0 1 1 0 0]
+
+            >>> # epsilon is 2 * m * ln((2 - f) / f) where m = 3 and f = .8
+            >>> print("epsilon:", m_rr.map(1))
+            epsilon: 2.4327906486489863
+
+We now convert a vector of randomized response bitvec releases 
+to an unbiased frequency estimate via ``debias_randomized_response_bitvec``:
+
+.. tab-set::
+
+    .. tab-item:: Python
+        :sync: python
+
+        .. code:: pycon
+
+            >>> frequencies = dp.m.debias_randomized_response_bitvec(
+            ...     [m_rr(data) for _ in range(10_000)],
+            ...     f=0.8,
+            ... )
```

Comment:
Doctest to at least show the shape of `frequencies`, even if we don't want to show the actual, random, results?

Reply:
Would it be good to have some conclusion on this in page in English after the code sample? Not a requirement.